package.json

```diff
@@ -35,6 +36,7 @@
     "@nuxtjs/google-fonts": "^2.0.0",
     "@nuxtjs/tailwindcss": "^6.1.3",
     "@tanstack/vue-query": "^4.19.1",
+    "axios": "^1.2.1",
```

Comment:
Ref: https://github.com/axodotdev/harbor/pull/43/files#r1042097528

Honestly I perfer axios in general even if it's not the way nuxt recommends axios is way more maintained, its more stable and its just super widely used so we know that if we have a problem someone has had it before